cumulus_library/template_sql/ctas_from_parquet.sql.jinja

```diff
@@ -0,0 +1,24 @@
+{%- import 'syntax.sql.jinja' as syntax -%}
+{%- if db_type == 'athena' -%}
+CREATE EXTERNAL TABLE IF NOT EXISTS `{{ schema_name }}`.`{{ table_name }}` (
+{%-  elif db_type == 'duckdb' -%}
+CREATE TABLE IF NOT EXISTS {{ table_name }} AS SELECT  
+{%- endif %}
+{%- for col in table_cols %}
+    {{ col }}{% if db_type == 'athena' %} {{ remote_table_cols_types[loop.index0] }}{%- endif -%}
+    {{- syntax.comma_delineate(loop) }}
+{%- endfor %}   
+{%- if db_type == 'athena' %}
+)
+{#- TODO: we may want to consider an optional partition parameter for
+large tables, though we would need to also run a MSCK REPAIR TABLE query 
+after this table is created to make the data available.
+
+See https://docs.aws.amazon.com/athena/latest/ug/parquet-serde.html
+for more info #}
+STORED AS PARQUET
+LOCATION '{{ remote_location }}'
+tblproperties ("parquet.compression"="SNAPPY");
```

Comment:
Do we need to specify compression? What happens if a researcher gives us a differently-compressed parquet file?

Reply:
we do, and it specifically needs to be this one and only this one. see [supported serdes](https://docs.aws.amazon.com/athena/latest/ug/supported-serdes.html). I think the deal is that the underlying serde may support multiple compressions, but AWS supports one. But this is about serializing the data from the parquet to the compressed table, not an artifact of the provided parquet itself.